bert_e/tests/test_bert_e.py

```diff
@@ -190,6 +190,66 @@ def dynamic_filtering(request):
 class QuickTest(unittest.TestCase):
     """Tests which don't need to interact with an external web services"""
 
+    def test_compare_branches_major_minor_vs_major_only(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, 3),)
+        branch2 = ((4, None),)
+        self.assertEqual(compare_branches(branch1, branch2), -1)
+
+    def test_compare_branches_major_only_vs_major_only_returns_0(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, None),)  # major-only
+        branch2 = ((4, None),)     # major.only
+        self.assertEqual(compare_branches(branch1, branch2), 0)
+
+    def test_compare_branches_major_only_vs_major_minor(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, None),)
+        branch2 = ((4, 3),)
+        self.assertEqual(compare_branches(branch1, branch2), 1)
+
+    def test_compare_branches_major_minor_micro_vs_major_minor(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, 3, 2),)
+        branch2 = ((4, 3),)
+        self.assertEqual(compare_branches(branch1, branch2), 2)
+
+    def test_compare_branches_major_minor_vs_major_minor_micro(self):
+        from bert_e.workflow.gitwaterflow.branches import compare_branches
+        branch1 = ((4, 3),)
+        branch2 = ((4, 3, 2),)
+        self.assertEqual(compare_branches(branch1, branch2), -2)
```

Comment:
Add those tests inside this file: https://github.com/scality/bert-e/blob/main/bert_e/tests/unit/test_sorted.py

and no need to keep importing `compare_branches` (already done in test_sorted at the top of the file)